src/ngraph_mark_for_clustering.cc

```diff
@@ -439,6 +440,9 @@ Status MarkForClustering(Graph* graph,
       // DT_FLOAT
       type_constraint_map["FusedBatchNormV2"]["T"] = {DT_FLOAT};
       type_constraint_map["FusedBatchNormGrad"]["T"] = NGraphNumericDTypes();
+      type_constraint_map["GatherNd"]["Tparams"] = {
+          DT_FLOAT};  // NGraphDTypes();
```

Comment:
Can we expand this to more types?

Reply:
TF python tests were throwing unsupported CPU builder errors for more types.

Reply:
TODO:   {DT_FLOAT} passes, but the full list `NGraphDTypes()` fails, so may need to check what other datatypes will pass